hash/msx1_cart.xml

```diff
@@ -18060,6 +18140,710 @@ legacy FM implementations cannot find it.
 
 <!-- Homebrew / Doujin software -->
 
+	<software name="o">
+		<description>O</description>
+		<year>2020</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Cobinee"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="o - cobinee (2020).rom" size="0x8000" crc="ae143f49" sha1="94f966c7bb15569edc87f6a0e971f1c5d8f46253" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="olion">
+		<description>Olion</description>
+		<year>2018</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Tiny Yarou"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x6000">
+				<rom name="olion - tiny yarou (2018).rom" size="0x6000" crc="ee3a0554" sha1="987c709827ae5730dc765b93676383426ebe1d50" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="openwide">
+		<description>Open Wide!</description>
+		<year>2021</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Hakogame"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="konami_scc" />
+			<dataarea name="rom" size="0x20000">
+				<rom name="open wide - hakogame (2021).rom" size="0x20000" crc="286a11a6" sha1="6c31e335447f6e7e1be2e34286c495b6cbef696c" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="opwolf">
+		<description>Operation Wolf (v3)</description>
+		<year>2006</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Toybox"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="0"/>
+			<dataarea name="rom" size="0xc000">
+				<rom name="operation wolf - toybox (2006) [msx-dev06 v3].rom" size="0xc000" crc="94a0f409" sha1="6287c1f05ea958e1abe86b87de073bef50977478" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="opwolfv1" cloneof="opwolf">
+		<description>Operation Wolf (v1)</description>
+		<year>2006</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Toybox"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="operation wolf - toybox (2006) [msx-dev06 v1].rom" size="0x8000" crc="e9492c87" sha1="366cc928a5dd5948bf6cf2ab1e00efcb2b6a8fd8" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="othelcp">
+		<description>Othello Competicao (Portuguese)</description>
+		<year>2003</year>
+		<publisher>&lt;homebrew&gt;</publisher>
```

Comment:
Shouldn’t this be “Competição”?

Reply:
Yes, you'd expect that in Portuguese. But when starting the software it displays "Othello competicao" (as can be seen in the screenshot from ICEknigh7.
Which is strange anyway since the international and Brazilian character sets do have ç and Ç as can be seen here: https://en.wikipedia.org/wiki/MSX_character_set

Reply:
It doesn’t seem to use “ç” in any of the text.  Very odd.